src/account/Settings.tsx

```diff
@@ -411,12 +416,33 @@ export const Account = ({ navigation, route }: Props) => {
             <SettingsItemCta
               title={t('keylessBackupSettingsTitle')}
               onPress={
-                cloudBackupCompleted ? onPressDeleteKeylessBackup : onPressSetUpKeylessBackup
+                deleteKeylessBackupStatus === KeylessBackupDeleteStatus.InProgress
+                  ? () => {
+                      // do nothing
+                    }
+                  : cloudBackupCompleted
+                    ? onPressDeleteKeylessBackup
+                    : onPressSetUpKeylessBackup
               }
               testID="KeylessBackup"
-              ctaText={cloudBackupCompleted ? t('delete') : t('setup')}
-              ctaColor={cloudBackupCompleted ? colors.error : colors.primary}
-              showChevron={!cloudBackupCompleted}
+              cta={
+                <>
+                  {deleteKeylessBackupStatus === KeylessBackupDeleteStatus.InProgress && (
+                    <LoadingSpinner width={32} />
```

Comment:
In the loading state the setting item resizes a bit not sure if this is expected.